### PR TITLE
Friends locations proper rendering

### PR DESCRIPTION
**Pull request by Manuhcuartas**
I finally managed to make it work, the way it is done is the following:
1. Everytime a new user enters the application and the LoMap folder is created (The one we use in order to manage all the data related with the application). It creates an empty locations.ttl file where all the locations should be stored. 
2. Once the folder and the file are both created it sets the permissions to append and read to all the friends in the SOLID friends list of the user.
3. Once all the before commented is set up, everytime the friends of such user login in the application they should be able to see their friends added locations without a problem.

### CODE REVIEW

**Review by RaduSeba — APPROVED** [review]
Everything seems to be fully working.

File: webapp/src/handlers/podAccess.js
Comment (juanmglzs) [inline webapp/src/handlers/podAccess.js:1]: getThing() method should be removed

Comment (FdezAriasSara) [reply webapp/src/handlers/podAccess.js:1]: When i run the app i got this:
image


I merged it with master to check if it was up to date and it was 
@juanmglzs do you get that too?

**Comment by FdezAriasSara** [comment]
I deleted the duplicate of writeLocations method in my local copy and it did run 

the thing is, my locations are not loading( nor are manus, which is my pod friend) 

im assuming this is because the new folder structure?? @Manuhcuartas  
Everything else seems to be working..


https://user-images.githubusercontent.com/79209608/233859280-98ad61b9-5e12-4d4e-ad9a-ed24257c6dde.mp4

**Review by FdezAriasSara — APPROVED** [review]
It works after deleting the repeated writeLocations method!

proof

https://user-images.githubusercontent.com/79209608/233860285-7b6dd3ea-0078-41d4-bde1-7db12fda484f.mp4

**Pull request merged**